bach/tests/functional/bach/test_series_string.py

```diff
@@ -110,3 +110,24 @@ def test_get_dummies(engine) -> None:
             [3, 1, 0, 0]
         ],
     )
+
+
+def test_string_replace(engine) -> None:
+    bt = get_df_with_test_data(engine)
+    municipality = bt['municipality'].sort_index()
+    assert isinstance(municipality, SeriesString)
+
+    municipalities = municipality.to_numpy()
```

Comment:
```suggestion
    municipalities = ['Leeuwarden', 'Súdwest-Fryslân', 'Súdwest-Fryslân']
```
Doing `to_numpy()` involves an extra query, which takes ~1.5 seconds for BigQuery. Considering in how many places we have hardcoded the expected values already, I don't think it's bad to do here, and it does safe run-time (and personally, I find it more readable actually)

Reply:
Done here https://github.com/objectiv/objectiv-analytics/pull/953/commits/110bcaa49477d0797201c744fa268b548198d474